<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widgetLeft" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="pathL"/>
         </item>
         <item>
          <widget class="QPushButton" name="changePathL">
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameL">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Permitions</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>User</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelSizeL">
                  <property name="text">
                   <string>Size</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDateL">
                  <property name="text">
                   <string>Date</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelNameL">
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="leftFiles"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetRight" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="pathR"/>
         </item>
         <item>
          <widget class="QPushButton" name="changePathR">
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameR">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="labelNameR">
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="labelSizeR">
                  <property name="text">
                   <string>Size</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDateR">
                  <property name="text">
                   <string>Date</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="rightFiles"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRename"/>
    <addaction name="actionMove"/>
    <addaction name="actionCreate_directory"/>
    <addaction name="separator"/>
    <addaction name="actionCat"/>
    <addaction name="actionCreate_file"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionCat">
   <property name="text">
    <string>Read file</string>
   </property>
  </action>
  <action name="actionCreate_directory">
   <property name="text">
    <string>Create directory</string>
   </property>
  </action>
  <action name="actionCreate_file">
   <property name="text">
    <string>Create file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
